docs/portal/transports/add-transport.md

```diff
@@ -11,6 +11,7 @@ The <strong> Transports </strong> page can be reached simply by clicking "Transp
 | Transport Name       | Enter a name that helps you identify this transport, if you leave it empty, the HDT portal will generate a default name in the format of "Target Domain:Target Port" |
 | Version Number       | The version number of the transport configuration in the history. |
```

Comment:
Please consider removing "The" at the beginning of this sentence.

Reply:
Will update it